alan/coo-997-new-docs-page-multilingual-websites-translating-consent (#27)
IdentitySquare/CookieChimp-Docs: docs/features/multilingual-banners.mdx

---
title: "Multilingual Banners"
description: "Translate your consent banner into different languages."
---

## Language Detection

We can detect language based on:
1. The HTML document language - controlled by the [lang attribute](https://www.tpgi.com/using-the-html-lang-attribute/) in the `html` element.
2. The user's browser language - set by the user's browser.

<img src="/images/consent-banner/language-detection.png" alt="Language Detection" />

> We recommend keeping the default language as English so we can always show default text in the banner if the visitor's language is not supported.

Reviewer: ```suggestion
This default language is used if the visitor's language is not supported or there is no translation available for the visitor's language.
```
